src/main/java/uk/gov/companieshouse/api/strikeoffobjections/service/impl/EmailService.java

```diff
@@ -55,29 +53,83 @@ public void sendObjectionSubmittedCustomerEmail (
             String companyNumber,
             Objection objection
     ) throws ServiceException {
+
         CompanyProfileApi companyProfile = companyProfileService.getCompanyProfile(companyNumber, requestId);
 
         String companyName = companyProfile.getCompanyName();
+        Map<String, Object> data = constructEmailDataMap(companyName, companyNumber, objection);
+
         String emailAddress = ericHeaderParser.getEmailAddress(ericAuthorisedUser);
-        Map<String, Object> data = new HashMap<>();
+        EmailContent emailContent = constructEmailContent(EmailType.CUSTOMER,
+                requestId, emailAddress, data);
 
-        data.put("company_name", companyName);
-        data.put("company_number", companyNumber);
-        data.put("objection_id", objection.getId());
-        data.put("reason", objection.getReason());
-        data.put("attachments", objection.getAttachments());
+        logger.debugContext(requestId, "Calling Kafka client to send customer email");
+        kafkaEmailClient.sendEmailToKafka(emailContent);
+        logger.debugContext(requestId, "Successfully called Kafka client");
+    }
+
+    @Override
+    public void sendObjectionSubmittedDissolutionTeamEmail(
+            String requestId,
+            String companyNumber,
```

Comment:
you can remove company number from the input params, it is on the Objection

Reply:
Removed